tests/cli/helpers/elastic_output.py

```diff
@@ -22,43 +22,46 @@ class ElasticSearchOutputArgumentsHelperTest(
   # pylint: disable=no-member,protected-access
 
   _EXPECTED_OUTPUT = """\
-usage: cli_helper.py [--index_name INDEX_NAME] [--doc_type DOCUMENT_TYPE]
-                     [--flush_interval FLUSH_INTERVAL] [--raw_fields]
-                     [--elastic_user ELASTIC_USER]
-                     [--elastic_password ELASTIC_PASSWORD] [--use_ssl]
-                     [--ca_certificates_file_path CA_CERTIFICATES_FILE_PATH]
-                     [--elastic_url_prefix ELASTIC_URL_PREFIX]
-                     [--server HOSTNAME] [--port PORT]
+usage: cli_helper.py [--index_name NAME] [--doc_type DOCUMENT_TYPE]
+                     [--flush_interval INTERVAL] [--raw_fields]
+                     [--elastic_mappings PATH] [--elastic_user USERNAME]
+                     [--elastic_password PASSWORD] [--use_ssl]
+                     [--ca_certificates_file_path PATH]
+                     [--elastic_url_prefix URL_PREFIX] [--server HOSTNAME]
+                     [--port PORT]
 
 Test argument parser.
 
 optional arguments:
-  --ca_certificates_file_path CA_CERTIFICATES_FILE_PATH
+  --ca_certificates_file_path PATH, --ca-certificates-file-path PATH
                         Path to a file containing a list of root certificates
                         to trust.
-  --doc_type DOCUMENT_TYPE
+  --doc_type DOCUMENT_TYPE, --doc-type DOCUMENT_TYPE
```

Comment:
Since ES 7, doc_type has been deprecated. We should consider removing it from Plaso as well. Let's do that in a followup PR though. Maybe add a TODO here?

Reply:
I've created an isssue https://github.com/log2timeline/plaso/issues/3415, thx for flagging